pkg/sysregistries/system_registries.go

```diff
@@ -97,3 +97,16 @@ func GetInsecureRegistries(sys *types.SystemContext) ([]string, error) {
 	}
 	return config.Registries.Insecure.Registries, nil
 }
+
+// RegistriesConfPath is the path to the system-wide registry configuration file
+func RegistriesConfPath(ctx *types.SystemContext) string {
+	dirPath := systemRegistriesConfPath
```

Comment:
(Non-blocking: BTW I don’t know why this was called `dirPath` when it does not point to a directory. No need to qualify it really, `path` would work just as well in this small function. Or, well, `return` and no variable.)

Reply:
Fixed